README.md

```diff
@@ -58,9 +57,9 @@ Then you can write your plugin specification in Lua, e.g. (in `~/.config/nvim/lu
 ```lua
 -- This file can be loaded by calling `lua require('plugins')` from your init.vim
 
-require('packer').startup({
+require('packer').add{
```

Comment:
The comment mentions `require`, but the text before (correctly) recommends putting this under `/plugin`. 

I guess the comment should be removed?

Reply:
Where does it say to put this under `/plugin`?

Reply:
🤦 

Classic reading failure. I read this as `.config/nvim/plugin`

Reply:
But my point still stands: just recommend putting this under `plugin`; no reason to perpetuate `require` shenanigans.

Reply:
Fair enough.

To note thatough that `:packadd!` behaves differently depending on if it's called in `init` vs `plugin`.

But for the most part, doing this in `plugin` should be fine.